fdbclient/include/fdbclient/TenantEntryCache.actor.h

```diff
@@ -44,8 +44,14 @@
 using TenantNameEntryPair = std::pair<TenantName, TenantMapEntry>;
 using TenantNameEntryPairVec = std::vector<TenantNameEntryPair>;
 
-enum class TenantEntryCacheRefreshReason { INIT = 1, PERIODIC_TASK = 2, CACHE_MISS = 3, REMOVE_ENTRY = 4 };
-enum class TenantEntryCacheRefreshMode { PERIODIC_TASK = 1, NONE = 2 };
+enum class TenantEntryCacheRefreshReason {
+	INIT = 1,
+	PERIODIC_TASK = 2,
+	CACHE_MISS = 3,
+	REMOVE_ENTRY = 4,
+	WATCH_TRIGGER = 5
+};
+enum class TenantEntryCacheRefreshMode { PERIODIC_TASK = 1, NONE = 2, WATCH = 3 };
```

Comment:
nit: Would move NONE to the last, given the state is in-memory, changing the ordering is OK, but keeping `none` as the last elements, assist in keeping `asserts` easy to add/extend.